synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -195,25 +195,21 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
 	}
 
 	String ffmpeg_binary_path;
-	std::list< String > binary_choices;
-	binary_choices.push_back("ffmpeg");
-	binary_choices.push_back("avconv");
-	std::list< String >::iterator iter;
-	for(iter=binary_choices.begin();iter!=binary_choices.end();iter++)
-	{
+	std::list<std::string> binary_choices = {"ffmpeg", "avconv"};
+	for(const auto& iter : binary_choices) {
 		String binary_path;
 #if defined(WIN32_PIPE_TO_PROCESSES)
 		binary_path = synfig::get_binary_path("");
-		if (binary_path != "")
+		if (!binary_path.empty()) {
 			binary_path = etl::dirname(binary_path)+ETL_DIRECTORY_SEPARATOR;
-		binary_path += *iter+".exe";
-		if( access( binary_path.c_str(), F_OK ) != -1 ) {
-			binary_path = "\"" + binary_path + "\"";
-			ffmpeg_binary_path = binary_path;
+		}
+		binary_path.append(iter).append(".exe");
+		if (g_access(binary_path.c_str(), F_OK ) != -1 ) {
```

Comment:
I imagine this is the issue solver? `g_access` instead of `access` ?

Reply:
Yes.